Voycar.Api.Web.Tests.Integration/Members/Get_Personal/Tests.cs

```diff
@@ -0,0 +1,86 @@
+namespace Voycar.Api.Web.Tests.Integration.Members.Get_Personal;
+
+using Context;
+using Microsoft.EntityFrameworkCore;
+using P = Features.Members.Endpoints.Get.Personal;
+
+
+public sealed class State : StateFixture
+{
+    public Guid MemberId { get; set; }
+}
+
+
+public class Tests : TestBase<App, State>
+{
+    private readonly App _app;
+    private readonly State _state;
+    private readonly VoycarDbContext Context;
+
+
+    // Setup request client
+    public Tests(App app, State state)
+    {
+        this._app = app;
+        this.Context = this._app.Context;
+        this._state = state;
```

Comment:
```suggestion
        this._state = state;
        this.Context = this._app.Context;
```
Reihenfolge bitte wie in der Klasse